packages/sources/bank-frick/src/config/index.ts

```diff
@@ -43,18 +44,36 @@ export const makeConfig = (prefix?: string): Config => {
     }
   }
 
-  let allowInsecure = false
-  if (process.env.ALLOW_INSECURE === 'true') {
-    if (process.env.NODE_ENV !== 'development') {
-      Logger.warn(
-        "ALLOW_INSECURE is true, but NODE_ENV isn't set to development. Ignoring the variable and setting ALLOW_INSECURE to false",
-      )
-    } else {
-      Logger.debug('ALLOW_INSECURE is true. Will skip certificate verification for requests')
-      allowInsecure = true
+  let privateKey = util.getRequiredEnv('PRIVATE_KEY')
```

Comment:
Not a huge deal, but we should probably still include the prefix here.

```suggestion
  let privateKey = util.getRequiredEnv('PRIVATE_KEY', prefix)
```